doc/storage_plugins.md

```diff
@@ -4,20 +4,183 @@ Storage plugins
 Content
 -------
   1. [Introduction](#introduction)
-  2. [Using Python](#using-python)
-  3. [Writing in C](#writing-in-c)
+  2. [Working with storages in Python](#working-with-storages-in-python)
+  3. [Using storages implicitely](#using-storages-implicitely)
+  4. [Writing Python storage plugins](#writing-python-storage-plugins)
+  5. [Working with storages from C and Fortran](#working-with-storages-from-c-and-fortran)
 
 
 Introduction
 ------------
-xxx
+A storage is in DLite an abstraction for an external data source or sink.
```

Comment:
What is meant by "an abstraction" for an external data source/sink?

Reply:
Ok, da har jeg la til noen setninger om det i avsnittet "Writing Python storage plugins"